ui/ts/models/log.ts

```diff
@@ -46,17 +48,14 @@ module Models {
         return "Local";
       };
 
-      private _data: Utils.QueryCache<Proto.LogEntry[]> = new Utils.QueryCache((): Promise<Proto.LogEntry[]> => {
-        return m.request({ url: this._url(), method: "GET", extract: nonJsonErrors })
-          .then((results: LogResponseSet) => {
-            return results.d;
-          });
-      });
+      private _data: Utils.QueryCache<Proto.LogEntry[]>;
 
       private _url(): string {
         let url: string = "/_status/logs/";
         if (this.node() != null) {
           url += this.node();
+        } else {
+          url += "local";
```

Comment:
if you go to https://localhost:8080/#/logs without a node id, it will show you the local logs. It used to work with no node id, but that's no longer the case since the updated .  Really it doesn't fit with the overall model of the ui but I didn't want to remove the feature yet.